themes/classic/_dev/css/theme.scss

```diff
@@ -312,10 +311,7 @@
       margin-bottom: 0.625rem;
       color: $gray-darker;
       .top-logo {
-        line-height: 50px;
-        vertical-align: middle;
-        width: 200px;
-        margin: 0 auto;
+        padding-top: 11px;
```

Comment:
This could create a problem. Not everyone is using a logo with the same dimensions as the default one and the 11px top padding does seem arbitrary to me.